templates/base.html.twig

```diff
@@ -4,10 +4,43 @@
         <meta charset="UTF-8">
         <title>{% block title %}Pet finder{% endblock %}</title>
         <link rel="icon" href="{{ asset("paw_favicon.png") }}" type="image/x-icon"/>
-        {% block stylesheets %}{% endblock %}
+        {% block stylesheets %}
+            <link href="{{ asset('petfinder.css') }}" rel="stylesheet" />
+        {% endblock %}
     </head>
     <body>
-        {% block body %}{% endblock %}
-        {% block javascripts %}{% endblock %}
+        <div class="header">
+            {% block header %}
+                <img class="header" src="{{ asset("photos/header.png") }}">
+            {% endblock %}
+        </div>
+        <div id="navigation">
+            {% block navigation %}
+                {% if is_granted('ROLE_USER') or is_granted('ROLE_ADMIN') %}
+                    <span><a href="/logout">logout</a></span>
```

Comment:
You should be using `paths` https://symfony.com/doc/current/templating.html#linking-to-pages for showing the url